src/ui/server/controllers/insurance/business/company-details/index.test.ts

```diff
@@ -46,6 +46,7 @@ describe('controllers/insurance/business/companies-details', () => {
           COMPANIES_HOUSE: `${INSURANCE_ROOT}/${mockApplication.referenceNumber}${COMPANY_HOUSE_SEARCH}`,
           COMPANY_DETAILS: `${INSURANCE_ROOT}/${mockApplication.referenceNumber}${COMPANY_DETAILS_ROUTE}`,
           NO_COMPANIES_HOUSE_NUMBER: `${INSURANCE_ROOT}/${mockApplication.referenceNumber}${NO_COMPANIES_HOUSE_NUMBER}`,
+          COMPANY_DETAILS_SAVE_AND_BACK: `${INSURANCE_ROOT}/${mockApplication.referenceNumber}${COMPANY_DETAILS_SAVE_AND_BACK}`,
```

Comment:
Can we call this `SAVE_AND_BACK_URL` to be consistent with others controllers?

Reply:
done